drop-transfer/src/transfer.rs

```diff
@@ -88,18 +88,58 @@ impl Transfer {
     }
 
     pub fn storage_info(&self) -> StorageInfo {
+        // TODO(msz): this insane check wouldn't be needed if we had two different
+        // `Transfer` types
+        let is_incoming = match self.files.values().next() {
+            Some(files) => matches!(files.kind, FileKind::FileToRecv { .. }),
+            None => true, // TODO(msz): Arbitrarily chosen, there is no way to differentiate here
```

Comment:
Not a very nice hack. Would be great to refactor the transfers. Mainly now there's some confusion as Transfers contain `files` that might be Incoming or Outgoing but then files themselves contain `kind` which is also `FileToSend` and `FileToRecv`. I will create a ticket for it